aws/ci_build

```diff
@@ -155,8 +155,13 @@ def main
         ChatClient.message 'server operations', commit_url, color: 'gray', message_format: 'text'
         DevelopersTopic.set_dtp 'yes'
         InfraProductionTopic.set_dtp_commit GitHub.sha('production')
-        zendesk_reminder = "/remind <@#{DevelopersTopic.dotd}> to check Zendesk in 2 hours"
-        ChatClient.message 'developers', zendesk_reminder
+        # You can find the channel id at the end of the url for a given channel using
+        # Slack in the browser, e.g https://codedotorg.slack.com/messages/<channel_id>
+        developers_channel_id = "C0T0PNTM3"
+        command = "remind"
+        zendesk_reminder = "<@#{DevelopersTopic.dotd}> to check Zendesk in 2 hours"
+        Slack.command developers_channel_id,
```

Comment:
Why call slack directly here instead of going through the more general chat client? 

What about adding a `ChatClient.set_reminder` method, which takes a room, message, and time span and formats the slack command? Especially since `command` is an undocumented API, we shouldn't expose the internals. If we set the channel ids in a settings hash like I suggested above, then we can internally look that up by name (and raise an error if it's missing). Then it would be something like (in ChatClient):
```ruby
def set_reminder(room, reminder)
```
Called as:
```ruby
ChatClient.set_reminder 'developers', '<@#{DevelopersTopic.dotd}> to check Zendesk in 2 hours'
```

or we could even make it more semantic:
```ruby
def set_reminder(room, recipient, message, time)
```
Called as:
```ruby
ChatClient.set_reminder 'developers', DevelopersTopic.dotd, 'check Zendesk', '2 hours'
```

Reply:
Side note: Rails has a bunch of nice time functions and helpers we could also use, but that might be getting more complex than necessary. For instance `2.hours` is an `ActiveSupport.Duration`. We could accept a duration or time and selectively format the command `in #{duration.to_s}` or `at #{time}`.